include/libdnf/base/goal.hpp

```diff
@@ -69,7 +66,18 @@ class Goal {
     void add_rpm_upgrade(const libdnf::rpm::Package & rpm_package);
     void add_rpm_upgrade(const libdnf::rpm::PackageSet & package_set);
 
-    bool resolve();
+    bool resolve(bool allow_erasing);
```

Comment:
What does return value mean? The seems that `true` means problems. It's alright?